lib/screens/create_region_screens/choose_region_id/components/authentication_status.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/design/app_colors.dart';
+
+enum AuthenticationState { loading, valid, invalid }
+
+class AuthenticationStatus extends StatelessWidget {
```

Comment:
auth status? I dont understand this

Reply:
@gguij004  Since this is a local component could you renamed to **RegionNameStatusIcon** I can counfuse this when I do seraching for authentication files.

Reply:
Change to RegionIdStatusIcon, thanks Raul for the suggestion